apps/btshell/src/cmd.c

```diff
@@ -368,7 +369,13 @@ cmd_advertise_start(int argc, char **argv)
         return rc;
     }
 
-    rc = ble_gap_ext_adv_start(instance, duration, max_events);
+    restart_adv = parse_arg_bool_dflt("restart_adv", 0, &rc);
```

Comment:
nitpick: I'd call this just `restart` since it's kind of obvious that you will restart advertising. and it's faster to type ;-)